### DISCUSSION TITLE
New badge

### DISCUSSION THREAD

**Discussion by Chamindu77** [community/community#116896] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

How to get a new badge on github?

**Comment by Duk3Dum0ntF4n** [comment, marked as answer]
If u speak about YOLO u need to add contributor to your project. After what create a new branch. Change something(update readme for example). Then create a pull request there a new contributor will be in reviewers group. After that just merge pull request by yourself
